app/controllers/paging_schedule_controller.rb

```diff
@@ -5,7 +5,9 @@
 ###
 class PagingScheduleController < ApplicationController
   rescue_from PagingSchedule::ScheduleNotFound do
-    render plain: 'Schedule not found', status: :not_found
+    turbo_frame = helpers.content_tag('turbo-frame', style: 'display:none', id: 'earliestAvailable') { 'No delievery estimate' }
+    turbo_frame += helpers.content_tag('body') { 'Schedule not found' }
+    render html: turbo_frame, layout: false, status: :not_found
```

Comment:
When reviewing, we should make sure the current estimated delivery date widget is ok with this change. It may not be turbo-aware (I'm uncertain if we can do `respond_to` in a `rescue_from` or not).